### PR TITLE
#1827: Improve AdminMediaGalleryUploadCategoryImageTest

### LINKED ISSUE #1827: Improve AdminMediaGalleryUploadCategoryImageTest
AdminMediaGalleryUploadCategoryImageTest fails sometimes on attempt to select `category` folder when `catalog` folder is not expanded

AdminMediaGalleryUploadCategoryImageTest should expand catalog folder (instead of selecting) before selecting category folder

### PR DESCRIPTION
**Pull request by joweecaquicla**
### Description (*)

This PR will introduce the improved/modified `AdminMediaGalleryUploadCategoryImageTest`.
### Fixed Issues (if relevant)

1. Fixes https://github.com/magento/adobe-stock-integration/issues/1827

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by joweecaquicla** [comment]
@magento run Functional Tests EE, Functional Tests CE, Functional Tests B2B

File: app/code/Magento/MediaGalleryUi/Test/Mftf/Test/AdminMediaGalleryUploadCategoryImageTest.xml [hunk 1]
```diff
@@ -37,9 +37,8 @@
         <actionGroup ref="AdminSaveCategoryFormActionGroup" stepKey="saveCategoryForm"/>
         <actionGroup ref="AdminOpenMediaGalleryFromCategoryImageUploaderActionGroup" stepKey="openMediaGalleryFromImageUploader"/>
         <actionGroup ref="ClearFiltersAdminDataGridActionGroup" stepKey="clearFilter"/>
-        <actionGroup ref="AdminMediaGalleryFolderSelectActionGroup" stepKey="selectCatalogFolder">
-            <argument name="name" value="catalog"/>
-        </actionGroup>
+        <conditionalClick selector="{{AdminMediaGalleryFolderSection.folderArrow(catalog)}}"
```
Comment (sivaschenko) [inline app/code/Magento/MediaGalleryUi/Test/Mftf/Test/AdminMediaGalleryUploadCategoryImageTest.xml]: Can you please move this conditional click to an action group (i.e.  `AdminMediaGalleryExpandFolder`)

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Thanks for  the pull request @joweecaquicla ! Please see my comment

**Comment by joweecaquicla** [comment]
@magento run Functional Tests EE, Functional Tests CE, Functional Tests B2B

**Review by sivaschenko — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko, thank you for the review.
[ENGCOM-8218](https://jira.corp.magento.com/browse/ENGCOM-8218) has been created to process this Pull Request

**Pull request merged**